### ISSUE TITLE
"auto_choose=False" is not working

### ISSUE THREAD

**Issue by fuatu** [digi604/django-smart-selects#242]
```
	tur= models.CharField(max_length=20, verbose_name="Tür",choices=TUR_CHOICES)
	borc = ChainedForeignKey(
		Borc, chained_field = 'tur',  chained_model_field = 'tur', blank=True, null=True,
		show_all=False, auto_choose=False, sort=True, 
		verbose_name="Borc",related_name="borc")
```
Even auto_choose is false when there is one record it selects the record automatically both in template and in django admin.

**Comment by czechmate247** [comment]
I don't know if this is the exact fix for this issue but I found a code modification that seems to be working for me.  I changed the following function in 'chainedfk.js' file:

`$.each(j, function (index, optionData) {
                        var option = $('<option></option>')
                            .prop('value', optionData.value)
                            .text(optionData.display);
                        if (auto_choose || init_value && optionData.value == init_value) {
                            option.prop('selected', true);
                        }
                        options.push(option);
                    });`

To the function below.  I found via browser developer console that "true" or "false" would sometimes be a boolean and sometimes be a string which would change the logic for this function:

`$.each(j, function (index, optionData) {
                        var option = $('<option></option>')
                            .prop('value', optionData.value)
                            .text(optionData.display);
                        if (auto_choose == "false" || auto_choose == "False"){
                            auto_choose = false;
                        } else if (auto_choose == "true" || auto_choose == "True"){
                            auto_choose = true;
                        }
                        if (auto_choose || init_value && optionData.value == init_value) {
                            option.prop('selected', true);
                        }
                        options.push(option);
                    });`

I hope this helps the bug fixing efforts.

**Comment by czechmate247** [comment]
You are most welcome and thank you for implementing this so quickly!  I will not be able to check your branch for a few days because I am away from my office.  I don’t want to keep you from submitting the changes though so please feel free!  Thanks again.

**Comment by blag** [comment]
Released to PyPI and GitHub in version `1.5.4`. Thanks!
